test/controllers/employer.test.ts

```diff
@@ -96,6 +96,36 @@ describe('user controller', () => {
     expect(response.body.id).toBe(1)
   })
 
+  it('should get employers', async () => {
+    await createEmployer()
+    const response = await request(app)
+      .get('/employer')
+
+    expect(response.status).toBe(200)
+  })
+
+  it('should update employer', async () => {
+    await createEmployer()
+    const employerId = await request(app)
+      .get('/employer/:id')
```

Comment:
The param should be a number: id of that newly created employer. Ex: `.get('/employer/1)`

Reply:
Thanks. I changed it to a number, but the test is still failing.